apps/web/app/(all)/[workspaceSlug]/(projects)/projects/(detail)/[projectId]/issues/(detail)/[issueId]/page.tsx

```diff
@@ -1,64 +1,68 @@
 "use client";
 
-import { useEffect } from "react";
-import { observer } from "mobx-react";
-import { useParams } from "next/navigation";
 import { useTheme } from "next-themes";
-import useSWR from "swr";
+import { redirect } from "react-router";
 import { useTranslation } from "@plane/i18n";
+// assets
+import emptyIssueDark from "@/app/assets/empty-state/search/issues-dark.webp?url";
+import emptyIssueLight from "@/app/assets/empty-state/search/issues-light.webp?url";
 // components
 import { EmptyState } from "@/components/common/empty-state";
 import { LogoSpinner } from "@/components/common/logo-spinner";
 // hooks
 import { useAppRouter } from "@/hooks/use-app-router";
-// assets
-import emptyIssueDark from "@/public/empty-state/search/issues-dark.webp";
-import emptyIssueLight from "@/public/empty-state/search/issues-light.webp";
 // services
 import { IssueService } from "@/services/issue/issue.service";
+// types
+import type { Route } from "./+types/page";
 
 const issueService = new IssueService();
 
-const IssueDetailsPage = observer(() => {
-  const router = useAppRouter();
-  const { t } = useTranslation();
-  const { workspaceSlug, projectId, issueId } = useParams();
-  const { resolvedTheme } = useTheme();
+export async function clientLoader({ params }: Route.ClientLoaderArgs) {
```

Comment:
@prateekshourya29 I would hold off on using clientLoader's and keep business logic the same, if there is something you're struggling with I can help, but we should minimize as much churn as possible.